src/test/ui/rfc-2632-const-trait-impl/call-const-trait-method.rs

```diff
@@ -0,0 +1,30 @@
+#![allow(incomplete_features)]
+#![feature(const_trait_impl)]
+#![feature(const_fn)]
+
+pub trait Plus {
+    fn plus(self, rhs: Self) -> Self;
+}
+
+impl const Plus for i32 {
```

Comment:
Would this test work the same if the `Add` trait were used instead of `Plus`?

Reply:
Yes. I'll add a type with a const impl of `Add` to this test. Obviously I can't do this directly on `i32` since that impl already exists. However, with the current implementation, it's possible to have const unstable trait impls even though it's not possible to have unstable ones. I will try to find an approach to trait solving that preserves this property.

Reply:
> Note that, with the current implementation, it's possible to have const unstable trait impls even though it's not possible to have unstable ones. This means we could have const unstable implementations of arithmetic traits for the primitive types. I will try to find an approach to trait solving that preserves this property.

Oh wow, that would be a neat feature